sshutil/sshutil.go

```diff
@@ -1,59 +1,113 @@
 package sshutil
 
 import (
-	"bytes"
-	"crypto/rand"
-	"crypto/rsa"
-	"crypto/x509"
+	"math/rand"
+	"crypto/ed25519"
 	"encoding/pem"
+	"bytes"
 
 	"golang.org/x/crypto/ssh"
 )
 
+// SSHKey stores byte slices that represent private and public ssh keys
 type SSHKey struct {
 	Private []byte
 	Public  []byte
 }
 
+// NewPrivatePublicKeyPair generates a an ed25519 ssh private and public key
 func NewPrivatePublicKeyPair(bits int) (SSHKey, error) {
```

Comment:
This `bits` value is no longer used so it should be removed

Reply:
Even better 😄 !